autorandr.py

```diff
@@ -638,7 +638,8 @@ def get_fb_dimensions(configuration):
         if "off" in output.options or not output.edid:
             continue
         # This won't work with all modes -- but it's a best effort.
-        o_width, o_height = map(int, output.options["mode"].split("x"))
+        o_mode = re.search("^[0-9]+x[0-9]+", output.options["mode"]).group(0)
```

Comment:
See #122, this needs to be more generic. How about

```suggestion
        o_mode = re.search("[0-9]{3,}x[0-9]{3,}", output.options["mode"]).group(0)
```
I doubt that this pattern will match anything but resolutions.